IntroToCSharp/Pages/Activities/Variables.razor

```diff
@@ -0,0 +1,148 @@
+                        important for the computer to know what type of data it is handling.
+                    </MudText>
+                    <Example Title="Ice Cream Truck">
+                        You are not going to go up to the ice cream truck and just say you want ice cream
+                        because the vender won’t know what flavor to give you. Instead, you tell him what flavor or type
+                        of ice cream you want. That way he can go straight to the freezer and get strawberry ice cream.
+                    </Example>
+                </MudListItem>
+            </NestedList>
+        </MudListItem>
+        <MudListItem Icon="@Icons.Material.Filled.Storage" IconColor="Color.Tertiary" Text="Data">
+            <NestedList>
+                <MudListItem>
+                    <MudText>Data is the actual information stored.</MudText>
+                    <Example Title="Fun Fact">
+                        Let's say that you memorize the date when the United States declared its
+                        independence. The "data" stored in your brain would be "July 4, 1776".
+                    </Example>
+                </MudListItem>
+            </NestedList>
+        </MudListItem>
+    </MudList>
+
+</Section>
+
+<Section Title="Using Variables">
+
+    <Note>We will learn what Console.ReadLine() and Console.WriteLine() do later.</Note>
+    <CodeBlock Filename="Variables/Test.cs" ReplIt="JoshuaJoshJosh1/VariablesExample" />
+    <MudText>
+        In order to declare a variable, you must first list the type (Ex: string, int, double, char, etc.) and then the
+        chosen name of the variable. Once you have declared a variable, you can use the variable's name in an expression. 
+        If you do not declare your variables beforehand, the program will not function.
+    </MudText>
+    <Note>
+        For proper formatting, variables are usually are placed at the very beginning of the program
+    </Note>
+    <MudText>
+        When writing a program, you will have many variables. Each one can be initialized only once, but can be
+        declared/reused
+        numerous times.
+    </MudText>
+</Section>
+
+<Section Title="More on Naming Variables">
+    <SubSection Title="The Rules">
+        <InfoGrid>
+            <GridSquare>Variable names must start with a
+                <b>letter</b>(a-z)
+                or <b>underscore</b>(_).
+            </GridSquare>
+            <GridSquare>You can't use numeric digits <b>0 through 9</b> to start your name.
+            </GridSquare>
+            <GridSquare>Many <b>symbols</b> and <b>blank spaces</b> will cause
+                the computer to not know where the variable ends.
+            </GridSquare>
+            <GridSquare>Variables cannot have the same name as a keyword like <b>"string"</b>.
+            </GridSquare>
+        </InfoGrid>
+    </SubSection>
+    <SubSection Title="Coding Guidlines for Naming Variables">
```

Comment:
The rules above are enforced by the C# compiler. That is to say, you cannot violate them. This still leaves a lot of room for naming your variables in ways that can be confusing. Remember, as a programmer, your job is to manage complexity. To help with that, here are some guidelines to help you give your variables "good" names: